src/org/omegat/gui/editor/EditorController.java

```diff
@@ -1880,6 +1880,17 @@ public void insertTag(final String tag) {
         }
     }
 
+    public void selectSourceText() {
```

Comment:
Because this is public method, so please add javadoc comment.

You can check CI test warning
```
[ant:checkstyle] [WARN] /home/vsts/work/1/s/src/org/omegat/gui/editor/EditorController.java:1883:5: Class 'EditorController' looks like designed for extension (can be subclassed), but the method 'selectSourceText' does not have javadoc that explains how to do that safely. If class is not designed for extension consider making the class 'EditorController' final or making the method 'selectSourceText' static/final/abstract/empty, or adding allowed annotation for the method. [DesignForExtension]

```